### DISCUSSION TITLE
BD Rhapsody VDJ v2 & VDJNext beads

### DISCUSSION THREAD

**Discussion by edyta-kowalczyk** [milaboratory/mixcr#1759] [Q&A]
Is the pipeline able to process the v2 VDJ and v3 VDJNext Rhapsody datasets?

**Comment by mizraelson** [comment]
Hi,
We have a dedicated preset for V2 beads: `bd-sc-xcr-rhapsody-full-length-enhanced-bead-v2`. However we didn't have a chance yet to work with V3 data. Do you know if the barcodes structure and whitelists are the same?

**Reply by mizraelson** [reply]
Hi, I just wanted to provide an update on this. I reached out to BD, and it seems that the barcode structure didn’t change in v3, so the preset `bd-sc-xcr-rhapsody-full-length-enhanced-bead-v2` should work as is. I would appreciate your feedback if you could give it a try.
